book/scheduling.md

```diff
@@ -505,7 +503,8 @@ class JSContext:
 ```
 
 The `__runXHROnload` method just pulls the relevant object from
-`XHR_REQUESTS` and calls its `onload` function:
+`XHR_REQUESTS` and calls its `onload` function, which is the standard
+callback for asynchronous `XMLHttpRequest`s:
```

Comment:
Otherwise we never mention what `onload`'s role is.

Reply:
What do you mean? I don't see why it's unclear.

Reply:
When we introduce XHR we should an example of how it's used, but that example is synchronous so it doesn't use `onload`. We never actually show an example with `onload` so someone who is not familiar with the XHR API might not understand what `onload` is or be missing the API to register an event handler.

Reply:
Ok. Not critical IMO